packages/block-library/src/template-part/index.php

```diff
@@ -114,7 +114,11 @@ function render_block_core_template_part( $attributes ) {
 	$is_debug = WP_DEBUG && WP_DEBUG_DISPLAY;
 
 	if ( is_null( $content ) ) {
-		if ( $is_debug && isset( $attributes['slug'] ) ) {
+		if ( $is_debug ) {
+			if ( ! isset( $attributes['slug'] ) ) {
+				// If there is no slug this is a placeholder and we dont want to return any message.
+				return '';
+			}
```

Comment:
I guess this check should be done outside of `$is_debug`. And only send the error message in debug mode.